backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -87,7 +87,7 @@ void SurfaceSdlGraphicsManager::setupScreen(uint gameWidth, uint gameHeight, boo
 	if (_fullscreen)
 		sdlflags |= SDL_FULLSCREEN;
 
-	_screen = SDL_SetVideoMode(effectiveWidth, effectiveHeight, 16, sdlflags);
```

Comment:
Just in case there was a misunderstanding, I tried to preserve 16 bits support smoothly (ie, support for 2 and 4 bytes pixel sizes), so on a pure "does it display garbage in 16 bits" level, 16 bits is still supported.
On the 16 bits performance level, in my tests in 16 bits mode I typically saw 16 bits at the same speed as without that patch (although I did not test all rebases).

This said, I do not know how widespread 16 its is among target systems, and if rare enough among devs that code path will get broken eventually. I can imagine 3D-capable-enough systems will be also capable of 24bits colors.